include/wx/dcsvg.h

```diff
@@ -66,7 +66,8 @@ class WXDLLIMPEXP_CORE wxSVGFileDCImpl : public wxDCImpl
 {
 public:
     wxSVGFileDCImpl( wxSVGFileDC *owner, const wxString &filename,
-                     int width=320, int height=240, double dpi=72.0 );
+                     int width = 320, int height = 240, double dpi = 72.0,
+                     const wxString &title = wxT("") );
```

Comment:
Hardly very important but it's better to use `= wxString()` for the default value (in spite of the fact that most of the existing code doesn't do it).